bin/v-update-letsencrypt-ssl

```diff
@@ -89,6 +89,17 @@ for user in $($HESTIA/bin/v-list-sys-users plain); do
             # Write the filtered alias list to the default var
             aliases=$f_aliases
 
+            domain_redirect="$(get_object_value 'web' 'DOMAIN' "$domain" '$REDIRECT')"
+            if [[ "$domain_redirect" != '' ]] ; then
+            	domain_redirect_code="$(get_object_value 'web' 'DOMAIN' "$domain" '$REDIRECT_CODE')"
```

Comment:
On line 70: https://github.com/hestiacp/hestiacp/blob/7eb78c045fe8d8cc0af1ba2c95e968f4a4164d43/bin/v-update-letsencrypt-ssl#L69-L70

source is allready used to source the code. So $REDIRECT_CODE, $REDIRECT and $SSL_FORCE should already be available to be used